package.json

```diff
@@ -61,6 +62,7 @@
   },
   "dependencies": {
     "@wordpress/autop": "^1.0.6",
+    "@wordpress/compose": "^1.0.1",
```

Comment:
why is this included as a dependency here, but then is [blacklisted here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/80/files#diff-41fdc10e1b5862877e4afa7e9176ee09R16)?

Reply:
OK, so, Metro being the smart tool it is, it actually tries to resolve the packages by looking in the local filesystem. So, when you _do_ want to use the remote package from NPM you need to blacklist the local package dir.